chain-impl-mockchain/src/txbuilder.rs

```diff
@@ -289,3 +289,136 @@ impl TransactionFinalizer {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::fee::LinearFee;
+    use crate::transaction::Input;
+    use crate::transaction::NoExtra;
+    use crate::transaction::INPUT_PTR_SIZE;
+    use chain_addr::Address;
+    use quickcheck::Arbitrary;
+    use quickcheck::Gen;
+    use std::ops::RangeBounds;
+
+    quickcheck! {
+        fn tx_builder_creates_balanced_tx(arbitrary_builder: ArbitraryTransactionBuilder) -> Result<(), &'static str> {
```

Comment:
I was speaking of this: https://docs.rs/quickcheck/0.8.5/quickcheck/struct.TestResult.html